runtime/jvmti/jvmtiStackFrame.c

```diff
@@ -57,11 +57,24 @@ jvmtiGetStackTrace(jvmtiEnv* env,
 
 		rc = getVMThread(currentThread, thread, &targetThread, TRUE, TRUE);
 		if (rc == JVMTI_ERROR_NONE) {
-			vm->internalVMFunctions->haltThreadForInspection(currentThread, targetThread);
+#if JAVA_SPEC_VERSION >= 19
+			if (NULL == targetThread) {
+				j9object_t threadObject = J9_JNI_UNWRAP_REFERENCE(thread);
+				if (IS_VIRTUAL_THREAD(currentThread, threadObject)) {
+					j9object_t contObject = (j9object_t)J9VMJAVALANGVIRTUALTHREAD_CONT(currentThread, threadObject);
+					rc = jvmtiInternalGetStackTrace(env, currentThread, NULL, contObject, start_depth, (UDATA) max_frame_count, frame_buffer, &rv_count);
```

Comment:
I don't see the `rc` from `jvmtiInternalGetStackTrace()` been checked. The original code didn't do that, but technically it should be checked.

Reply:
No special handling is required. In case of an error, the subsequent code to `resumeThreadForInspection`, `releaseVMThread` and `internalExitVMToJNI` needs to be executed. After that, `rc` is correctly returned from `jvmtiGetStackTrace` by `TRACE_JVMTI_RETURN`.

Reply:
OK.